samples/interceptors/wso2-micro-integrator/docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3.7"
+services:
+  legacy-xml-backend:
+    image: wso2am/cc-sample-legacy-xml-backend:v1.0.0
+    ports:
+      - "9080:9080"
+    networks:
+      - choreo-connect_default
+  xml-interceptor:
+    image: wso2am/cc-sample-xml-interceptor-wso2-mi:v1.0.0
+    ports:
+      - "9081:8253"
```

Comment:
Since we are using a different port `8253` and if it is not possible to change the port, we have to add a description on changing the port in the sample API OpenAPI spec that asks to change the port of interceptor service defined in `x-wso2-interceptor`.

Reply:
Added a note at the end of the same `README.md` file.